rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -109,6 +110,20 @@ const FlowEditor: React.FC<{
   }>({});
   const isDragging = useRef(false);
 
+  const [tutorialStarted, setTutorialStarted] = useState(false); // State to control if tutorial has started
```

Comment:
Can this be a component that takes children rather than making this MORE complex?

Reply:
I don't see many changes in `Flow.tsx`, only this state and one `useEffect` but if you want you can make a custom hook for this, most of the tutorial is inside `tutorial.tsx`.